### ISSUE TITLE
Let's resolve conflict with $(body).removeClass('...')

### ISSUE THREAD

**Issue by Firfi** [MohammadYounes/AlertifyJS#86]
Hey @MohammadYounes ,

recently I found an issue with scrolling: after close() on modal, page scrolling didn't worked.

Reason was in `ajs-no-overflow` not being deleted. I debugged code and found that your removeClass() function is really called, but doesn't delete `ajs-no-overflow` class, moreover, this class appears at the start of class="" string in body like `class="ajs-no-overflow home"`. 

Reason appeared to be jQuery addClass() and removeClass() conflict: my application calling it _before_ your modal code calling removeClass(), therefore if looks like this:

before modal called:

`landing`

after modal called:

`landing ajs-no-overflow`

after body class changed with `$(body).removeClass('landing')`

`ajs-no-overflow` (sic! no space anymore as jQuery truncate it)

after body class changed with `$(body).addClass('home')`

`ajs-no-overflow home`

after modal .close() is called and therefore your removeClass() is called:

`ajs-no-overflow home` (sic! is is still there as removeClass() expect a space)

I propose to change removeClass() implementation to regex one described here: `http://stackoverflow.com/questions/195951/change-an-elements-class-with-javascript`

```
className.replace
      ( /(?:^|\s)MyClass(?!\S)/g , '' )
```

This way conflict with jQuery will be resolved.

I don't think changing app logic will be good way either; it is possible that users will call jquery addClass/removeClass by different reasons and will have this issue therefore.

I'll make pull request without build (npm install to get grunt takes forever it seems) soon.

Will do it in master as it isn't a new feature but rather bugfix.

**Comment by MohammadYounes** [comment]
Thanks @Firfi for reporting this 

Using a regex in this case will be much slower than normal arrays, I suggest the following fix:

```js
function removeClass(element, classNames) {
  var original = element.className.split(' ');      
  var toBeRemoved = classNames.split(' ');
          
  for (var x = 0; x < toBeRemoved.length; x += 1) {
    var index = original.indexOf(toBeRemoved[x]);
    if (index > -1)
      original.splice(index,1);
  }
  element.className = original.join(' ');
}
``` 

Update the PR and I'll release the fix soon.

Thanks!

**Comment by Firfi** [comment]
Hey @MohammadYounes ,

added your implementation, tested it. 

I added 4-space span there to conform library code

Thanks!

**Comment by MohammadYounes** [comment]
Released in 1.6.1 ... Thank you!
